src/platform/crypt_openssl.c

```diff
@@ -59,25 +145,73 @@ CxPlatCryptInitialize(
         return QUIC_STATUS_TLS_ERROR;
     }
 
+#ifdef IS_OPENSSL_3
     //
-    // OPENSSL_init_ssl() may leave errors in the error queue while returning
-    // success.
+    // Preload ciphers
     //
+    if (!CxPlatLoadCipher("AES-128-GCM", &CXPLAT_AES_128_GCM_ALG_HANDLE) ||
+        !CxPlatLoadCipher("AES-256-GCM", &CXPLAT_AES_256_GCM_ALG_HANDLE) ||
+        !CxPlatLoadCipher("AES-128-ECB", &CXPLAT_AES_128_ECB_ALG_HANDLE) ||
+        !CxPlatLoadCipher("AES-256-ECB", &CXPLAT_AES_256_ECB_ALG_HANDLE) ||
+        !CxPlatLoadCipher("ChaCha20-Poly1305", &CXPLAT_CHACHA20_POLY1305_ALG_HANDLE))
+        goto Error;
```

Comment:
Our coding style requires `{` and `}` around all code blocks.
```suggestion
        !CxPlatLoadCipher("ChaCha20-Poly1305", &CXPLAT_CHACHA20_POLY1305_ALG_HANDLE)) {
        goto Error;
    }
```
Please update the other code too.

Reply:
Done hopefully.